src/main/java/seedu/address/logic/commands/RepaidCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands;
```

Comment:
not really a clean pr?

Reply:
The title of the PR is "....and sync with current commands". I think it is a clean PR law hmmm.

Reply:
could have had repaid command implementation in a separate pr before working on whitelist

Reply:
Repaid Command is not a current command. it is new.

Reply:
Yeap you are right guys. It could have been in another PR to reduce complications. Sorry for the trouble guys.